include/seastar/core/fair_queue.hh

```diff
@@ -380,14 +378,6 @@ public:
 
     void update_shares_for_class(class_id c, uint32_t new_shares);
 
-    /// \return how many waiters are currently queued for all classes.
-    [[deprecated("fair_queue users should not track individual requests, but resources (weight, size) passing through the queue")]]
```

Comment:
for ease of review, it was deprecated in 7a5b29ccc9086fc68e0dcf418ef4c6e5f27d2c9b